src/EditorFeatures/Test2/Expansion/NameExpansionTests.vb

```diff
@@ -222,7 +222,7 @@ class C
             Await TestAsync(input, expected, expandParameter:=True)
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.Expansion)>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/11979"), Trait(Traits.Feature, Traits.Features.Expansion)>
```

Comment:
> I don't have enough context to attempt a fix.

I think we need to have a plan for addressing this before merging this change. It is generally not Ok to break an existing behavior. Having an issue opened is not a plan. 

---
In reply to: [67024193](https://github.com/dotnet/roslyn/pull/11981#discussion_r67024193) [](ancestors = 67024193,67016763,67015720)